### PR TITLE
8.4: fix e2e tests

### PR DESCRIPTION
**Pull request by skurfuerst**
This change fixes the TestCafe Setup

### CODE REVIEW

**Comment by skurfuerst** [comment]
Hm, I do not understand why the e2e tests still fail on CircleCI. Locally they run through now. @markusguenther @mhsdesign  any ideas? :) 

ERROR Cannot find the browser. "saucelabs:Firefox@latest:Windows 10" is neither a known browser alias, nor a path to an executable file.

( https://app.circleci.com/pipelines/github/neos/neos-ui/4693/workflows/a1011688-8c49-4a16-a12f-0dda25b22982/jobs/21390/artifacts )

All the best,
Sebastian

**Review by grebaldi — APPROVED** [review]
Hi @skurfuerst,

thanks a lot for taking care of this :heart: I tested both `make test-e2e-docker` and `make start-neos-dev-instance` locally and they both work perfectly :+1: 

@skurfuerst:
> ERROR Cannot find the browser. "saucelabs:Firefox@latest:Windows 10" is neither a known browser alias, nor a path to an executable file.

Since you didn't touch CI, I suspect the error is on saucelabs' end (Wasn't there some EOL announcement lately?). We probably need to revise our CI strategy for the E2E tests, so I'd say it's fine to ignore the red check as far as this PR is concerned.

**Comment by skurfuerst** [comment]
Thanks @grebaldi for testing and giving feedback <3 <3

File: packages/neos-ui-editors/src/Editors/Image/index.js [hunk 1]
```diff
@@ -72,7 +72,9 @@ export default class ImageEditor extends Component {
                 isAssetLoading: true
             }, () => {
                 this.loadImage = loadImageMetadata(this.props.value.__identity).then(image => {
-                    this.setState({image, isAssetLoading: false});
+                    if (this._isMounted) {
```
Comment (mhsdesign) [inline packages/neos-ui-editors/src/Editors/Image/index.js:75]: this probably fixes the flacky cropping tests? I also experienced issues here ^^

File: README.md [hunk 2]
```diff
@@ -161,14 +161,19 @@ To setup end-to-end tests locally you have got to do the same things described i
...
 To speed up the e2e-test workflow/feedback loop you can start the system under test in a docker setup and run the tests against that:
-* `make start-neos-dev-instance` (starts a docker setup with the system under test)
+* `make start-neos-dev-instance` (starts a docker setup with the system under test and keep it running - in the `1Dimension` scenario)
 * The neos dev instance is available at `localhost:8081`
 * To enter the container run `docker compose -f Tests/IntegrationTests/docker-compose.neos-dev-instance.yaml exec php bash`
 * `yarn run testcafe <browser> <testFile> <optional flags>`
-  * for example, this runs all tests in chrome:
-  `yarn run testcafe chrome Tests/IntegrationTests/Fixtures`
+  * for example, this runs all tests in chrome: (NOTE starting with Chrome 127, --disable-search-engine-choice-screen is needed)
+  `yarn run testcafe chrome:--disable-search-engine-choice-screen Tests/IntegrationTests/Fixtures/1Dimension`
```
Comment (mhsdesign) [inline README.md:176]: thanks for the `--disable-search-engine-choice-screen` hint its really stupid of chrome :D

**Review by mhsdesign — COMMENTED** [review]
Post merge +1 :)

Thanks for updating our testcafe from 2.x to 3.x 🥳

**Comment by skurfuerst** [comment]
Will upmerge today

**Pull request merged**